frontend/src/scenes/insights/EditorFilters/EFPathsAdvanced.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react'
+import { useActions, useValues } from 'kea'
+import { AvailableFeature, EditorFilterProps, PathEdgeParameters } from '~/types'
+import { InfoCircleOutlined, SettingOutlined } from '@ant-design/icons'
+
+import { PayCard } from 'lib/components/PayCard/PayCard'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+import { pathsLogic } from 'scenes/paths/pathsLogic'
+import { InputNumber, Tooltip } from 'antd'
+import { Link } from 'lib/components/Link'
+import { PathCleanFilterInput } from '../InsightTabs/PathTab/PathCleanFilterInput'
+
+export function EFPathsAdvanced({ insightProps }: EditorFilterProps): JSX.Element {
```

Comment:
This component seems kind of cramped, though happy to defer to another PR.
<img width="476" alt="image" src="https://user-images.githubusercontent.com/53387/167384723-02b5fea5-aff5-4635-8d9f-16ef47bfd5ac.png">